packages/reports/addon/components/dimension-selector.js

```diff
@@ -69,56 +63,12 @@ export default class DimensionSelector extends Component {
     return [...dimensions, ...timeDimensions];
   }
 
-  /*
-   * @property {Object} defaultTimeGrain - the default time grain for the logical table selected
-   */
-  @computed('request.tableMetadata.timeGrains')
-  get defaultTimeGrain() {
-    return getDefaultTimeGrain(this.request.tableMetadata.timeGrains);
-  }
-
-  /*
-   * @property {Array} listItems - all list items to populate the dimension selector,
-   *                               combination of timegrains and dimensions
-   */
-  @computed('allTimeGrains', 'allDimensions')
-  get listItems() {
-    // only option is to add the default time grain (if none is selected)
-    const timeGrain = {
-      name: 'Date Time',
-      category: 'Date',
-      dateTimeDimension: true
-    };
-
-    return [timeGrain, ...this.allDimensions];
-  }
-
   /*
    * @property {Array} selectedDimensions - dimensions in the request
    */
   @computed('request.dimensionColumns.[]')
-  get selectedDimensions() {
-    return this.request.dimensionColumns;
-  }
-
-  /*
-   * @property {Object} selectedTimeGrain - timeGrain in the request
-   */
-  @computed('request.timeGrainColumn')
-  get selectedTimeGrain() {
-    return this.request.timeGrainColumn;
-  }
-
-  /*
-   * @property {Object} selectedColumns - unique selectedDimensions
-   */
-  @computed('selectedTimeGrain', 'selectedDimensions')
   get selectedColumns() {
-    if (!this.selectedTimeGrain) {
-      return this.selectedDimensions;
-    } else {
-      return arr([this.selectedTimeGrain, ...this.selectedDimensions]).uniq();
-    }
+    return this.request.dimensionColumns;
   }
 
   /*
```

Comment:
Can we rename the `itemsChecked` prop?

Reply:
I can combine `selectedColumns` and `itemsChecked` and call it `selectedColumns`.